.github/workflows/release.yaml

```diff
@@ -763,16 +763,18 @@ jobs:
         if: failure()
         uses: 8398a7/action-slack@v3
         with:
-          status: ${{ job.status }}
+          status: custom
           fields: repo,ref,workflow,job,took
           custom_payload: |
             {
               attachments: [{
+                color: '${{ job.status }}' === 'success' ? 'good' : '${{ job.status }}' === 'failure' ? 'danger' : 'warning',
                 text: `${process.env.AS_WORKFLOW}\n${process.env.AS_JOB} of ${process.env.AS_REPO}@${process.env.AS_REF}\n${{ job.status }} in ${process.env.AS_TOOK}`,
               }]
             }
         env:
           SLACK_WEBHOOK_URL: ${{ secrets.KOTS_BUILD_STATUS_SLACK_WEBHOOK_URL }}
+          MATRIX_CONTEXT: ${{ toJson(matrix) }}
```

Comment:
this is necessary when jobs include the matrix field. however, it's apparently buggy and the docs and GH issues are very vague on when and why it will fail. proposing we try it out and delete if it's still not working.